processing file asmFiles/test.branchjump1.asm (1 of 5)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.ldst1.asm (2 of 5)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.ldst2.asm (3 of 5)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.loadstore.asm (4 of 5)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.rtype.asm (5 of 5)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.branchjump1.asm (1 of 5)
--- memsim.hex	2022-02-20 17:46:46.608475362 -0500
+++ memcpu.hex	2022-02-20 17:46:46.544472951 -0500
@@ -13,6 +13,8 @@
 :04000C00AC41000CF7
 :04000D0017E3FFF9FD
 :04000E00FFFFFFFFF2
-:0400220000000018C2
-:0400240000000030A8
+:040020000000BA5CC6
+:040021000000BA5CC5
+:0400220000000024B6
+:040023000000BA5CC3
 :00000001FF
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
--- memsim.hex	2022-02-20 17:46:52.162684650 -0500
+++ memcpu.hex	2022-02-20 17:46:52.096682163 -0500
@@ -13,7 +13,7 @@
 :040020000000733732
 :0400210000002701B3
 :040022000000133790
-:04002300DEADBEEFA1
+:040023000000BEEF2C
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
processing file asmFiles/test.rtype.asm (5 of 5)
--- memsim.hex	2022-02-20 17:46:54.096757528 -0500
+++ memcpu.hex	2022-02-20 17:46:54.025754852 -0500
@@ -35,7 +35,6 @@
 :04002500FFFFDA708F
 :04002600000037F8A7
 :04002700000021565E
-:04002800000D269011
-:04002900000006933A
-:04003C00FFFF0806B4
+:040028000000D26999
+:0400290000000D26A0
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2022-02-20 17:57:34.393884501 -0500
+++ memcpu.hex	2022-02-20 17:57:34.309881336 -0500
@@ -13,26 +13,26 @@
 :04000C00FFFFFFFFF4
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:040023000000008554
+:040024000000010DCA
+:04002500000001993D
+:0400260000000229AB
+:04002700000002BD16
+:04002800000003557C
+:04002900000003F1DF
+:04002A00000004913D
+:04002B000000053597
+:04002C00000005DDEE
+:04002D000000068940
+:04002E00000007398E
+:04002F00000007EDD9
+:04003000000008A51F
+:040031000000096161
+:0400320000000A219F
+:0400330000000AE5DA
+:0400340000000BAD10
+:0400350000000C7942
+:0400360000000D4970
+:0400370000000E1D9A
 :0403C0000000001623
 :00000001FF
